vtk-cdat/conda_build_config.yaml

```diff
@@ -0,0 +1,17 @@
+python:
+    - 3.6
+    - 2.7
+pin_run_as_build:
+  python: x.x
+
+ffmpeg:
+    - 4.0.*
+pin_run_as_build:
```

Comment:
nice one.

Reply:
Yes. I just tried it and it works! Initially Initially I tried it without pinning a version for ffmpeg (just left conda to pick up the latest version) but I could not make that work.